<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clinical_picture</class>
 <widget class="QWidget" name="clinical_picture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Признаковое описание заболеваний</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Left">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Right">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Доступные признаки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбранные признаки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeft">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRight_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLeft_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
